src/core/server/core_usage_stats/core_usage_stats_client.ts

```diff
@@ -0,0 +1,185 @@
+          total: current.total + 1,
+          kibanaRequest: incrementKibanaRequestCounter(current.kibanaRequest, headers),
+          createNewCopiesEnabled: incrementBooleanCounter(
+            current.createNewCopiesEnabled,
+            createNewCopies
+          ),
+          overwriteEnabled: incrementBooleanCounter(current.overwriteEnabled, overwrite),
+        },
+      },
+    };
+    await this.updateUsageStats(attributes);
+  }
+
+  public async incrementSavedObjectsResolveImportErrors({
+    headers,
+    createNewCopies,
+  }: IncrementSavedObjectsResolveImportErrorsOptions) {
+    const coreUsageStats = await this.getUsageStats();
+    const { apiCalls = {} } = coreUsageStats;
+    const {
+      savedObjectsResolveImportErrors: current = SAVED_OBJECTS_RESOLVE_IMPORT_ERRORS_DEFAULT,
+    } = apiCalls;
+
+    const attributes = {
+      ...coreUsageStats,
+      apiCalls: {
+        ...apiCalls,
+        savedObjectsResolveImportErrors: {
+          total: current.total + 1,
+          kibanaRequest: incrementKibanaRequestCounter(current.kibanaRequest, headers),
+          createNewCopiesEnabled: incrementBooleanCounter(
+            current.createNewCopiesEnabled,
+            createNewCopies
+          ),
+        },
+      },
+    };
+    await this.updateUsageStats(attributes);
+  }
+
+  public async incrementSavedObjectsExport({
+    headers,
+    types,
+    supportedTypes,
+  }: IncrementSavedObjectsExportOptions) {
+    const coreUsageStats = await this.getUsageStats();
+    const { apiCalls = {} } = coreUsageStats;
+    const { savedObjectsExport: current = SAVED_OBJECTS_EXPORT_DEFAULT } = apiCalls;
+    const isAllTypesSelected = !!types && supportedTypes.every((x) => types.includes(x));
+
+    const attributes = {
+      ...coreUsageStats,
+      apiCalls: {
+        ...apiCalls,
+        savedObjectsExport: {
+          total: current.total + 1,
+          kibanaRequest: incrementKibanaRequestCounter(current.kibanaRequest, headers),
+          allTypesSelected: incrementBooleanCounter(current.allTypesSelected, isAllTypesSelected),
+        },
+      },
+    };
+    await this.updateUsageStats(attributes);
+  }
+
+  private async updateUsageStats(attributes: CoreUsageStats) {
+    const options = { id: CORE_USAGE_STATS_TYPE, overwrite: true };
+    try {
+      await this.repository.create(CORE_USAGE_STATS_TYPE, attributes, options);
+    } catch (err) {
+      // do nothing
+    }
+  }
+}
+
+function incrementKibanaRequestCounter(current: { yes: number; no: number }, headers?: Headers) {
+  // The presence of these three request headers gives us a good indication that this is a first-party request from the Kibana client.
+  // We can't be 100% certain, but this is a reasonable attempt.
+  const isKibanaRequest = headers && headers['kbn-version'] && headers.origin && headers.referer;
+  return {
+    yes: current.yes + (isKibanaRequest ? 1 : 0),
+    no: current.no + (isKibanaRequest ? 0 : 1),
+  };
+}
```

Comment:
I added this function to CoreUsageStatsClient and the Spaces plugin's UsageStatsClient.

We attempt to differentiate whether a request originated from the Kibana client or not. There's no way to be 100% certain, but we can check if these three headers exist, which gives us a good approximation. Even if a third-party integration is using the `kbn-version` header, it is unlikely those requests also contain an `origin` and `referer` header too.